### ISSUE TITLE
Not sure what this error is, feel free to update this issue when someone figures it out

### ISSUE THREAD

**Issue by sbarbosadataverse** [IQSS/askdataverse#2]
I asked the data a question, and got this

<img width="1436" alt="Screen Shot 2024-02-26 at 12 31 58 PM" src="https://github.com/IQSS/askdataverse/assets/8322346/d1c95119-7357-4e6c-bcfe-5aeb889f7cc6">

**Comment by antonpolishko** [comment]
The error means that the data is too long for the context window for the LLM used. 

Either changes in chunking needed on the askdataverse bot side, or upgrade to model with bigger context window on API endpoint side.

**Comment by siacus** [comment]
correct, I need to change the endpoint for the LLM, I'll keep you posted.

**Comment by siacus** [comment]
This error should be gone after having changed the endpoint. Please re-open in case it appears again.
